ui/app/src/utils/content.ts

```diff
@@ -256,14 +256,14 @@ export function parseLegacyItemToDetailedItem(item: LegacyItem | LegacyItem[]):
       sizeInBytes: null
     },
     staging: {
-      lastScheduledDate: item.scheduledDate,
-      lastPublishedDate: item.publishedDate,
+      lastScheduledDate: item.scheduledDate ?? item.eventDate,
+      lastPublishedDate: item.publishedDate ?? item.eventDate,
```

Comment:
- `eventDate` can't be both. it's either `datePublished` or `dateScheduled`.
- These prop names are actually outdated. Please update `PublishEnvProps` interface
  -  lastScheduledDate => dateScheduled
  - lastPublishedDate => datePublished
<img width="421" alt="image" src="https://user-images.githubusercontent.com/3928341/126327073-1b699af8-3143-4fac-a43d-f36a55bc1767.png">